builders/integrations/indexers/moralis.md

```diff
@@ -0,0 +1,182 @@
+
+## Moralis SDK {: #moralis-SDK }
+
+Moralis has an SDK that allows developers to seamlessly integrate Moralis' API into their backend infrastructure. This SDK offers a wide array of features, including:
+
+- Data querying from both EVM and Solana APIs  
+- Integration of Web3 authentication  
+- A collection of utility functions for efficient data transformation and formatting  
+
+Moralis currently provides official SDK support for two primary programming languages:
+
+- NodeJS
+- Python
+
+To install Moralis SDK:
+
+=== "npm"
+
+    ```bash
+    npm install moralis
+    ```
+
+=== "yarn"
+
+    ```bash
+    yarn add moralis
+    ```
+
+=== "pnpm"
+
+    ```bash
+    pnpm add moralis
+    ```
+
+=== "pip"
+
+    ```bash
+    pip install moralis
+    ```
+
+Once the SDK is ready, you can leverage it to query Moralis APIs. Here's an example of how to interact with the EVM API using a JSON RPC interface:
+
+=== "Javascript"
+
+    ```js
+    import Moralis from 'moralis';
+
+        try {
+          await Moralis.start({
+            apiKey: "INSERT_YOUR_API_KEY"
+    });
+
+          const response = await Moralis.EvmApi.block.getBlock({
+            "chain": "0x504",
+            "blockNumberOrHash": "1"
+    });
+
+          console.log(response.raw);
+    } catch (e) {
+          console.error(e);
+    }
+    ```
+
+=== "Python"
+
+    ```python
+    from moralis import evm_api
+
+    api_key = "INSERT_YOUR_API_KEY"
+
+    params = {
+          "chain": "moonbeam",
+          "block_number_or_hash": "1"
+    }
+
+    result = evm_api.block.get_block(
+          api_key=api_key,
+          params=params,
+    )
+
+        print(result)
+    ```
+
+
+## Stream API {: #stream-api }
+
+Moralis Streams API is a tool for developers to listen to and react to events happening on blockchains in real time. This API lets you listen for specific events, such as a new transaction being added to a block, a particular smart contract function being called, or an NFT being transferred and delivering the event via Webhook.
+
+Head over to the Streams page within Moralis and click **Create a new stream** to get started.
+
+![stream API page](/images/builders/integrations/indexers/moralis/moralis-2.webp)
+
+To get started quickly, let's choose a predefined template here. In this example, we're interested in xcDOT transactions. Select the "Contract Activity" template. This will allow us to easily listen for events related specifically to xcDOT.
+
+![template selection page](/images/builders/integrations/indexers/moralis/moralis-3.webp)
+
+On the next page, enter the following details: 
+
+- Smart Contract Address for [xcDot](https://moonscan.io/token/0xffffffff1fcacbd218edc0eba20fc2308c778080): 0xFfFFfFff1FcaCBd218EDc0EbA20Fc2308C778080
+- Network: Select Moonbeam from the available options.
+Moralis supports listening to events on Moonbeam, Moonriver, and Moonbase Alpha.  
```

Comment:
Let's convert this to steps and align it with the steps they have on their UI. And then you can combine images 4-6 into a single image.

You should use snagit. I just put together some docs on how to use it. I'll share the page in slack